tensorflow/compiler/xla/service/gpu/gemm_rewriter.cc

```diff
@@ -373,6 +362,31 @@ class GemmRewriterVisitor : public DfsHloRewriteVisitor {
     return OkStatus();
   }
 
+  Status ExchangeBinaryOpAndSlice(HloInstruction *instr, HloInstruction *slice,
+                                  HloInstruction *broadcast) {
+    // Re-broadcast the bias or zero pattern to the shape of the GEMM.
```

Comment:
My intent is to help future readers understand the code we're writing here.  A point of confusion is that this function switches between levels of abstraction.  In the fn name and parameter names, we're talking about something very generic: An instruction, a slice, and a broadcast.  But then in this comment, we're talking about a "bias or zero pattern".  That's much less generic.

To help people understand, we should operate at a consistent level of abstraction.